packages/loaders/stories/3-Inline.stories.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { PALETTE } from '@zendeskgarden/react-theming';
+import { Inline, IInlineProps } from '@zendeskgarden/react-loaders';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+
+export default {
+  title: 'Components/Loaders/Inline',
+  component: Inline
+} as Meta;
+
+export const Default: Story<IInlineProps> = ({ color, size }) => {
+  return (
+    <Grid aria-busy="true" aria-live="polite">
+      <Row>
+        <Col md textAlign="center">
+          <Inline size={size} color={color} />
+        </Col>
+        <Col md textAlign="center">
+          <Inline size={size} color={color} />
+        </Col>
+        <Col md textAlign="center">
+          <Inline size={size} color={color} />
+        </Col>
+      </Row>
+    </Grid>
+  );
+};
+
+Default.argTypes = {
+  size: {
+    name: 'size (px)',
```

Comment:
The "(px)" probably not necessary as the units are clear from the prop description.